cluster/gce/container-linux/master-helper.sh

```diff
@@ -79,6 +79,11 @@ function create-master-instance-internal() {
     preemptible_master="--preemptible --maintenance-policy TERMINATE"
   fi
 
+  local min_cpu_selection=""
+  if [[ -n "${MASTER_MIN_CPU_ARCHITECTURE:-}" ]]; then
+    min_cpu_selection="--min-cpu-selection=${MASTER_MIN_CPU_ARCHITECTURE}"
```

Comment:
The example I saw used `--min-cpu-platform` not `--min-cpu-selection`?

Reply:
Good catch.. fixed it.